examples/jabberwockipedia/index.html

```diff
@@ -0,0 +1,3259 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Jabberwocky - Wikipedia</title>
```

Comment:
When it was `koi`, I named it `koi.html`. I suggest naming this file `jabberwockipedia.html`. It's easier to distinguish in the editor when you want to open the file. If all example files are named `index.html` then you have to pay attention to the folder in which they are, and I don't think the name actually matters for other things ?